bindings/python/slimt.cpp

```diff
@@ -204,7 +216,7 @@ PYBIND11_MODULE(_slimt, m) {
       .def(py::init<size_t, size_t>(), py::arg("workers") = 1,
            py::arg("cache_size") = 0)
       .def("translate", &PyService::translate, py::arg("model"),
-           py::arg("texts"), py::arg("html") = false)
+           py::arg("texts"), py::arg("html") = false, py::arg("encoding"))
```

Comment:
This one is missing default argument, which should be `py::arg("encoding") = Encoding.UTF8`. Does this not cause problems?

Reply:
https://github.com/jerinphilip/slimt/pull/38/commits/a31d578acb192abca542355cec6f0a410df28389